src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -234,6 +238,24 @@ const formConfig = {
           uiSchema: choosePtsdType.uiSchema,
           schema: choosePtsdType.schema,
         },
+        ptsdWalkthroughChoice781: {
+          title: 'PTSD Walkthrough 781 Choice',
+          path: 'new-disabilities/walkthrough-781-choice',
+          depends: formData =>
+            hasNewPtsdDisability(formData) && needsToEnter781(formData),
+          uiSchema: ptsdWalkthroughChoice781.uiSchema,
+          schema: ptsdWalkthroughChoice781.schema,
+        },
+        uploadPtsdDocuments781: {
+          title: 'Upload PTSD Documents - 781',
+          path: 'new-disabilities/ptsd-781-upload',
+          depends: formData =>
+            hasNewPtsdDisability(formData) &&
+            needsToEnter781(formData) &&
+            isUploadingPtsdForm(formData),
```

Comment:
Yes, a user won't be able to make the upload selection if `hasNewPtsdDisability` wasn't true on a previous page, but I feel like checking it here makes sense because it's explicit and makes the logic of the 781 flow more obvious.

We do have more pages coming that will make the need for `isUploadingPtsdForm` more necessary.